File: 백준/Bronze/2675. 문자열 반복/문자열 반복.js
function solution(input) {
    let result = '';
    for(let i = 1; i <= input[0]; i++) {
        const STRING_TO_ARRAY = input[i][1].split('');
        const ARRAY = STRING_TO_ARRAY.map(el => el.repeat(input[i][0]));
        result += ARRAY.join('') + '\n';
    }
    console.log(result);
}

const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,    
});

let input = [];
let list = [];

rl.on("line", function (line) {
   input.push(line)
}).on("close", function () {
    input.forEach((val) => {
       list.push(val.split(' ').map((el) => (el)));
    });
    solution(list);
    process.exit();
});

File: 백준/Bronze/2675. 문자열 반복/문자열 반복.cc
#include <iostream>
using namespace std;

int main()
{
    int t, n;
    string str, result;
    
    cin >> t;
    
    for (int i = 0; i < t; i++){
        cin >> n >> str;
        
        for (int j = 0; j < str.length(); j++) {
            for(int k = 0; k < n; k++) {
                result += str[j];
            }
        }
        result += '\n';
    }
    
    cout << result;
    return 0;
}